terminus-ui/cohort-date-range/src/cohort-date-range.component.md

```diff
@@ -0,0 +1,95 @@
+<h1>Cohort Date Range</h1>
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Basic usage](#basic-usage)
+- [Event driven](#event-driven)
+- [Inputs](#inputs)
+  - [allowCustomDates](#allowCustomDates)
+  - [Disable the component](#disable-the-component)
+
+## Basic usage
+
+Pass in the cohorts:
+
+```html
+<ts-cohort-date-range
+  [cohorts]="cohorts"
+></ts-cohort-date-range>
+```
+
+```typescript
+  public cohorts = [{
+    display: 'Last 90 days',
+    range: {
```

Comment:
I'd vote on not using 3rd party library code in our examples if possible. Maybe just use `new Date(x, x, x)` instead?